pkg/cfn/manager/create_tasks.go

```diff
@@ -81,6 +81,23 @@ func (c *StackCollection) NewUnmanagedNodeGroupTask(nodeGroups []*api.NodeGroup,
 	return taskTree
 }
 
+// DoAllNodegroupStackTasks iterates over nodegroup tasks and returns any errors.
+func (c *StackCollection) DoAllNodegroupStackTasks(taskTree *tasks.TaskTree, region, name string) error {
+	logger.Info(taskTree.Describe())
+	errs := taskTree.DoAllSync()
+	if len(errs) > 0 {
+		logger.Info("%d error(s) occurred and nodegroups haven't been created properly, you may wish to check CloudFormation console", len(errs))
+		logger.Info("to cleanup resources, run 'eksctl delete nodegroup --region=%s --cluster=%s --name=<name>' for each of the failed nodegroup", region, name)
+		for _, err := range errs {
+			if err != nil {
+				logger.Critical("%s\n", err.Error())
+			}
+		}
+		return fmt.Errorf("failed to create nodegroups for cluster %q", name)
+	}
+	return nil
+}
+
```

Comment:
does this need to live in the `manager` package? I think it could live in `pkg/actions/nodegroup`, since it just executes tasks it doesn't really belong to the stack manager

Reply:
sure! I don't see any issues with moving it, sounds good

Reply:
Moved it too!